Roslyn analyzers failing to load on Mac

When our build is run on a Mac machine the analyzers for our project will fail to load with the following error:

>CSC : warning CS8032: An instance of analyzer Microsoft.NetCore.Analyzers.Runtime.TestForNaNCorrectlyAnalyzer cannot be created from /Users/jaredpar/.nuget/packages/microsoft.netcore.analyzers/2.6.0-beta1-62231-02/analyzers/dotnet/cs/Microsoft.NetCore.Analyzers.dll : Exception has been thrown by the target of an invocation.. [/Users/jaredpar/code/roslyn/src/Test/Utilities/Portable/TestUtilities.csproj]
  TestUtilities -> /Users/jaredpar/code/roslyn/Binaries/Debug/Dlls/TestUtilities/netstandar

Disabling for now in order to get Mac builds running.

--- a/eng/build.sh
+++ b/eng/build.sh
@@ -226,13 +226,7 @@ function BuildSolution {
     bl="/bl:\"$log_dir/Build.binlog\""
   fi
   
-  local projects="$repo_root/$solution" 
-  
-  # https://github.com/dotnet/roslyn/issues/23736
-  UNAME="$(uname)"
-  if [[ "$UNAME" == "Darwin" ]]; then
-    run_analyzers=false
-  fi
+  local projects="$repo_root/$solution"
 
   # NuGet often exceeds the limit of open files on Mac and Linux
   # https://github.com/NuGet/Home/issues/2163

--- a/eng/build.sh
+++ b/eng/build.sh
@@ -228,6 +228,7 @@ function BuildSolution {
   
   local projects="$repo_root/$solution"
 
+  UNAME="$(uname)"
   # NuGet often exceeds the limit of open files on Mac and Linux
   # https://github.com/NuGet/Home/issues/2163
   if [[ "$UNAME" == "Darwin" || "$UNAME" == "Linux" ]]; then

